testsuite/src/test/java/org/eclipse/krazo/test/ext/Jsr223IT.java

```diff
@@ -69,8 +79,8 @@ public void testNashorn() {
     // in 2020 (EOL Python 2) we probably need to remove this test with Jython 2.7 anyway
     // and the above mentioned configuration effort would count for nothing.
     @Category(IgnoreOnWildfly.class)
-    public void testJython() {
-        webDriver.navigate().to(baseURL + "mvc/jython?name=Jython");
-        assertTrue(webDriver.getPageSource().contains("Hello Jython"));
+    public void testJython() throws Exception {
+        final HtmlPage page = webClient.getPage(baseURL + "mvc/jython?name=Jython");
+        assertTrue(page.getTextContent().contains("Hello Jython"));
```

Comment:
```java
java.lang.NullPointerException: null
	at org.eclipse.krazo.test.ext.Jsr223IT.testJython(Jsr223IT.java:84)
```

Reply:
Looks like the corresponding controller doesn't generate any HTML tags, just plain text content. Maybe that's the cause?